### ISSUE TITLE
[Bug] Endstone Smelter caused server crash

### ISSUE THREAD

**Issue by Zunwolf1896** [quiqueck/BetterEnd#35]
### What happened?

Title explained. 

### BetterEnd

2.0.6

### BCLib

2.0.6

### Fabric API

0.57.0

### Fabric Loader

0.14.7

### Minecraft

1.19

### Relevant log output

```shell
---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 7/10/22, 9:43 AM
Description: Exception in server tick loop

java.lang.ExceptionInInitializerError
	at org.betterx.betterend.blocks.entities.EndStoneSmelterBlockEntity.method_5465(EndStoneSmelterBlockEntity.java:234)
	at net.minecraft.class_2624.createMenu(class_2624.java:92)
	at net.minecraft.class_3222.method_17355(class_3222.java:1040)
	at org.betterx.betterend.blocks.EndStoneSmelter.openScreen(EndStoneSmelter.java:79)
	at org.betterx.betterend.blocks.EndStoneSmelter.method_9534(EndStoneSmelter.java:71)
	at net.minecraft.class_4970$class_4971.method_26174(class_4970.java:963)
	at net.minecraft.class_3225.redirect$bcg000$activateWithOptionalCactus(class_3225.java:3029)
	at net.minecraft.class_3225.method_14262(class_3225.java:338)
	at net.minecraft.class_3244.method_12046(class_3244.java:1115)
	at net.minecraft.class_2885.method_12547(class_2885.java:34)
	at net.minecraft.class_2885.method_11054(class_2885.java:8)
	at net.minecraft.class_2600.method_11072(class_2600.java:22)
	at net.minecraft.class_1255.method_18859(class_1255.java:157)
	at net.minecraft.class_1255.method_16075(class_1255.java:131)
	at net.minecraft.server.MinecraftServer.handler$ejg000$onWaitingForNextTick(MinecraftServer.java:13776)
	at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java)
	at net.minecraft.server.MinecraftServer.handler$zpi000$modifiedRunLoop(MinecraftServer.java:3889)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:644)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:258)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalStateException: Registry is already frozen (trying to add key ResourceKey[minecraft:menu / betterend:end_stone_smelter])
	at net.minecraft.class_2370.method_40274(class_2370.java:81)
	at net.minecraft.class_2370.method_31051(class_2370.java:91)
	at net.minecraft.class_2370.method_10273(class_2370.java:87)
	at net.minecraft.class_2370.method_10272(class_2370.java:135)
	at net.minecraft.class_2378.method_39197(class_2378.java:508)
	at net.minecraft.class_2378.method_10230(class_2378.java:504)
	at net.fabricmc.fabric.api.screenhandler.v1.ScreenHandlerRegistry.registerSimple(ScreenHandlerRegistry.java:90)
	at org.betterx.betterend.client.gui.EndStoneSmelterScreenHandler.<clinit>(EndStoneSmelterScreenHandler.java:29)
	... 20 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.19
	Minecraft Version ID: 1.19
	Operating System: Linux (amd64) version 4.18.0-348.7.1.el8_5.x86_64
	Java Version: 17.0.3, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 2999203040 bytes (2860 MiB) / 10737418240 bytes (10240 MiB) up to 10737418240 bytes (10240 MiB)
	CPUs: 2
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 5800X 8-Core Processor
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 0
	Microarchitecture: Zen 3
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: unknown
	Graphics card #0 vendor: unknown
	Graphics card #0 VRAM (MB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 81698.59
	Virtual memory used (MB): 89936.12
	Swap memory total (MB): 17407.99
	Swap memory used (MB): 4331.73
	JVM Flags: 20 total; -Xmx10240M -Xms6656M -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1
	Fabric Mods: 
		additionalbanners: AdditionalBanners 8.0.1
		additionalstructures: Additional Structures 4.0.0
		adorn: Adorn 3.5.4+1.19
		advancednetherite: Advanced Netherite 1.6.0-1.19
		alloy_forgery: Alloy Forgery 2.0.15+1.19
		animal_origins_18: Animal Origins 1.0.0
		antixray: AntiXray 1.2.2
			com_moandjiezana_toml_toml4j: toml4j 0.7.2
		appleskin: AppleSkin 2.4.0+mc1.19
		architectury: Architectury 5.8.29
		attributefix: AttributeFix 15.0.1
		awesomedungeonend: Awesome Dungeon Edition end 1.0.3
		awesomedungeonnether: Awesome Dungeon Edition Nether 1.0.3
		awesomedungeonocean: Awesome Dungeon Edition Ocean 2.0.8
		badpackets: Bad Packets 0.1.2
		banhammer: BanHammer 0.5.2+1.19
			club_minnced_discord-webhooks: discord-webhooks 0.8.0
			com_squareup_okhttp3_okhttp: okhttp 3.14.9
			com_squareup_okio_okio: okio 1.17.2
			fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
			mysql_mysql-connector-java: mysql-connector-java 8.0.29
			org_jetbrains_annotations: annotations 22.0.0
			org_json_json: json 20210307
			org_slf4j_slf4j-api: slf4j-api 1.7.32
			org_xerial_sqlite-jdbc: sqlite-jdbc 3.36.0.3
			sgui: sgui 1.1.0+1.19
		bclib: BCLib 2.0.12
		beaconoverhaul: Beacon Overhaul 1.7.0+1.19
			fabric-api-base: Fabric API Base 0.4.9+e62f51a3a9
			fabric-networking-api-v1: Fabric Networking API (v1) 1.0.27+7fe97409a9
			reach-entity-attributes: Reach Entity Attributes 2.3.0
		beo: Basic End Ores 1.19-3.0.0-Fabric
		better_bad_omen: Better Bad Omen 1.4.0
		betteranimalsplus: Better Animals Plus 1.19-11.0.6
		betterechoshards: Better Echo Shards 1.2
		betterend: Better End 2.0.6
		betternether: Better Nether 7.0.6
		bettervillage: Better Village 1.0.2
		bitsandchisels: BitsAndChisels 2.7.0
			stacc: Stacc 1.3.4
		blockus: Blockus 2.5.1+1.19
			terraform-wood-api-v1: Terraform Wood API (v1) 4.1.0
		bno: Basic Nether Ores 1.19-8.0.0-Fabric
		bookshelf: Bookshelf 14.0.3
		carpet: Carpet Mod 1.4.79
		cave_enhancements: Cave Enhancements 1.2.6
		charm: Charm 4.1.2
		chunky: Chunky 1.2.202
		cloth-config: Cloth Config v6 7.0.72
			cloth-basic-math: cloth-basic-math 0.6.1
		clumps: Clumps 9.0.0+3
		collective-fabric: Collective (Fabric) 4.27
			fiber: fiber 0.23.0-2
		cotw: Curse Of The Warden 1.0.0
		craftycuisine: Crafty Cuisine 1.1.0
		crawl: Crawl 0.11.0
		creeperoverhaul: Creeper Overhaul 2.0.0
		croptopia: Croptopia 2.0.6
			com_typesafe_config: config 1.4.1
			io_leangen_geantyref_geantyref: geantyref 1.3.11
			org_spongepowered_configurate-core: configurate-core 4.1.2
			org_spongepowered_configurate-hocon: configurate-hocon 4.1.2
		ctov: ChoiceTheorem's Overhauled Villages 3.0
		darkpaintings: DarkPaintings 11.0.2
		dcintegration-fabric: Discord Integration Fabric 2.4.5
			de_erdbeerbaerlp_dcintegration_common: dcintegration.common 2.4.5
		deathlog: DeathLog 0.2.10+1.19
		dramaticdoors: Dramatic Doors (Fabric) 1.19-1.2.0
		dual_riders: Dual Riders 1.1.2
		ecologics: Ecologics 2.1.3
		economical_villager_trading: Economical Villager Trading 1.2.6
			crowdin-translate: CrowdinTranslate 1.4+1.19
		eer: Erroring Entity Remover 1.0.3+1.19
		ellemes_container_lib: Ellemes' Container Library 2.0.1
		emerald_tools: Emerald Tools 1.4.16
			pugh_tools: Pugh Tools 1.3.10
		endrem: End Remastered 5.1.0
		essential_commands: Essential Commands 0.24.4-mc1.19
			com_fasterxml_jackson_dataformat_jackson-dataformat-yaml: jackson-dataformat-yaml 2.13.0
			ec-core: Essential Commands Core 1.0.1-mc1.19
			playerabilitylib: Pal 1.6.0
		expandedstorage: Expanded Storage 8.0.1
		experiencebugfix: Experience Bug Fix 18
		extradiscs: Extra Discs 1.0.2
		extrakeybinds: ThatRobins Extra Origins Keybinds 0.3.0
		extraorigins: Extra Origins 1.19-1
		fabric: Fabric API 0.57.0+1.19
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.7+9ff28f40a9
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.14+b2a4a624a9
			fabric-command-api-v1: Fabric Command API (v1) 1.2.6+f71b366fa9
			fabric-command-api-v2: Fabric Command API (v2) 2.1.2+0d55f585a9
			fabric-commands-v0: Fabric Commands (v0) 0.2.23+df3654b3a9
			fabric-containers-v0: Fabric Containers (v0) 0.1.27+df3654b3a9
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.2.0+07df213ea9
			fabric-convention-tags-v1: Fabric Convention Tags 1.0.8+37622d24a9
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.3+bd0a0d4aa9
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.1.1+a680b9b4a9
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.28+a6d2f785a9
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.16+9ff28f40a9
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.26+9ff28f40a9
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.25+df3654b3a9
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.19+18990361a9
			fabric-item-api-v1: Fabric Item API (v1) 1.5.4+767db26ea9
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.25+9ff28f40a9
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.1.0+33fbc738a9
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.0+03a4e568a9
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.3+9e7660c6a9
			fabric-message-api-v1: Fabric Message API (v1) 1.0.1+513f4a59a9
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.10+33fbc738a9
			fabric-networking-v0: Fabric Networking (v0) 0.3.14+df3654b3a9
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.0.7+9ff28f40a9
			fabric-particles-v1: Fabric Particles (v1) 1.0.8+dc39553aa9
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.17+23c4cfefa9
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.12+9ff28f40a9
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.5+9ff28f40a9
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.0.9+e62f51a3a9
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.5.5+a6d2f785a9
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.2.6+9ff28f40a9
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.0.9+e62f51a3a9
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.1.1+9e7660c6a9
		fabric-language-kotlin: Fabric Language Kotlin 1.8.1+kotlin.1.7.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.3
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.3
		fabricloader: Fabric Loader 0.14.8
		fabsit: FabSit 1.3.0+1.19
		fancyclear: FancyClear 1.3.3
		ferritecore: FerriteCore 4.2.1
		fishofthieves: Fish of Thieves 2.0.3
		flan: Flan 1.19-1.7.10
		flytre_lib: FlytreLib 1.8.8
		foxboi_two: Kitsune 1.1.4
		friendsandfoes: Friends&Foes 1.5.4
			mm: Manningham Mills 2.3
		fwaystones: Fabric Waystones 3.0.2+mc1.19
		geckolib3: Geckolib 3.1.5
			com_eliotlash_mclib_mclib: mclib 18
			com_eliotlash_molang_molang: molang 18
		grounded_origins: Grounded Origins 1.2.1
			apugli: Apugli 1.7.2
				com_github_llamalad7_mixinextras: MixinExtras 0.0.10
		guardvillagers: Guard Villagers Fabric 1.19-1.0.14
		herdspanic: Herds Panic 1.0.6
		hopobetterunderwaterruins: Hopo Better Underwater Ruins 1.0.7
		icarus: Icarus 1.13
		immersive_weathering: Immersive Weathering 1.0.1-1.19_SNAPSHOT
		inventorysorter: Inventory Sorter 1.8.8-1.19
			kyrptconfig: Kyrpt Config 1.4.6-1.19
				blue_endless_jankson: jankson 1.2.1
		invview: InvView 1.4.9-1.19+
		java: OpenJDK 64-Bit Server VM 17
		jei: Just Enough Items 11.0.0.206
		justplayerheads-fabric: Just Player Heads (Fabric) 2.1
		lazydfu: LazyDFU 0.1.3
		ledger: Ledger 1.2.5
			server_translations_api: Server Translations API 1.4.14+1.19-rc2
				packet_tweaker: Packet Tweaker 0.3.0+1.18.2
		ledger-databases: Ledger Databases 1.1.1
			com_h2database_h2: h2 1.4.200
			org_postgresql_postgresql: postgresql 42.3.0
		libraryferret: Library Ferret 2.0.0
		lithium: Lithium 0.8.1
		logprot: Logprot Mod 1.19-1.6
		luckperms: LuckPerms 5.4.28
		mavm: More Axolotl Variants Mod 1.1.0
		mcda: MC Dungeons Armors 2.1.1
		mcdw: MC Dungeons Weapons 4.4.0-1.19
			enchant_giver: Enchant Giver 1.3.0
		mcwbridges: Macaw's Bridges 2.0.3
		mcwdoors: Macaw's Doors 1.0.6
		mcwfences: Macaw's Fences and Walls 1.0.5
		mcwlights: Macaw's Lights and Lamps 1.0.3
		mcwpaintings: Macaw's Paintings 1.0.3
		mcwroofs: Macaw's Roofs 2.1.2
		mcwtrpdoors: Macaw's Trapdoors 1.0.6
		mcwwindows: Macaw's Windows 2.0.3
		medievalorigins: Medieval Origins 1.19.x-4.1.1
		memoryleakfix: memory Leak Fix 1.19-0.4.1
		midnightlib: MidnightLib 0.5.2
		minecraft: Minecraft 1.19
		mob_catcher: Mob Catcher 1.2.4
		moborigins: Mob Origins 1.9.0
		mobs_attempt_parkour: Mobs Attempt Parkour 0.3-1.19-pre1
			maelstrom_library: Maelstrom Library 1.3-1.19-pre1
		modid: Creatures Of The Snow 2.2.0+1.19
		moreblockbricks: More Block Bricks 1.4.0
		mythicmetals: Mythic Metals 0.15.1
			magna: Magna 1.8.0-1.19
				omega-config: OmegaConfig 1.0.8
		mythicmetals_decorations: Mythic Metals Decorations 0.3.0
		naturalist: Naturalist 2.0.1
		naturescompass: Nature's Compass 1.19-2.0.4-fabric
		nears: Nears 1.2.2
		nomobfarm: No Mob Farm 1.6.3
		origins: Origins 1.6.2
			apoli: Apoli 2.5.1
				additionalentityattributes: Additionalentityattributes 1.0.0+1.19
				calio: Calio 1.7.0
		owo: oωo 0.7.3+1.19
		oxidized: Oxidized 1.7.0
		paintings: Paintings++ 10.0.0.0
		pandamc_concreteplus: Concrete Plus 1.9.5
		patchouli: Patchouli 1.19-74-FABRIC
		pehkui: Pehkui 3.3.3+1.14.4-1.19.1
			kanos_config: Kanos Config 0.3.1+1.14.4-1.19
		phantomsseephantoms: Phantoms See Phantoms 1.0.0
		phosphor: Phosphor 0.8.1
		player_events: Player Events 2.3.1
			placeholder-api: Placeholder API 2.0.0-beta.6+1.19
			player_events_api: Player Events API 2.3.1
		pvptoggle: Pvp Toggle 1.0.6
		reinfchest: Reinforced Chests 2.0.3+1.19
			reinfcore: Reinforced Core 2.1.0+1.19
		rocks: This Rocks! 1.6.1
		rpgdifficulty: Rpg Difficulty 1.2.0
		short_grass: Short Grass 1.1.0
		spark: spark 1.9.13
		stackrefill-fabric: Stack Refill (Fabric) 2.2
		styledplayerlist: Styled Player List 2.2.1+1.19
			player-data-api: Player Data API 0.1.0+1.18
		terrablender: TerraBlender 2.0.0.117
			com_electronwill_night-config_core: core 3.6.4
			com_electronwill_night-config_toml: toml 3.6.4
		the_bumblezone: The Bumblezone - Fabric 6.0.1+1.19
			fake-player-api: Fake Player API 0.3.0
		toofast: TooFast 1.0.0
		transparent: Transparent 5.1.0
		trinkets: Trinkets 3.4.0
			cardinal-components-base: Cardinal Components API (base) 5.0.0-beta.1
			cardinal-components-entity: Cardinal Components API (entities) 5.0.0-beta.1
		unsaddle: Unsaddle 0.1.0+1.17
			unsaddle-core: Unsaddle Core 0.1.0
		vcinteraction: Voice Chat Interaction 1.19-1.0.1
		vmp: Very Many Players 0.2.0+beta.5.0
		voicechat: Simple Voice Chat 1.19-2.2.45
		whisperwoods: Whisperwoods 1.19-2.1.0
		worldedit: WorldEdit 7.2.11-beta-01+6200-0877589
		wthit: wthit 5.5.1
		xaerominimap: Xaero's Minimap 22.10.1
		xaeroworldmap: Xaero's World Map 1.24.0
		yigd: You're in Grave Danger 1.4.1
	Server Running: true
	Player Count: 9 / 1337; [class_3222['fincifer'/57078, l='ServerLevel[Zunwolf5]', x=13404.49, y=147.82, z=12620.58], class_3222['Zeatux'/60064, l='ServerLevel[Zunwolf5]', x=7247.02, y=64.00, z=-24350.93], class_3222['DragonSaka'/63291, l='ServerLevel[Zunwolf5]', x=-1109.72, y=116.00, z=-455.96], class_3222['tsukidragonshark'/68430, l='ServerLevel[Zunwolf5]', x=-27.87, y=7.52, z=43.08], class_3222['korpipat'/61636, l='ServerLevel[Zunwolf5]', x=7254.55, y=64.00, z=-24322.97], class_3222['RomanticWings66'/84901, l='ServerLevel[Zunwolf5]', x=13424.04, y=143.94, z=12618.82], class_3222['MasterXado'/86178, l='ServerLevel[Zunwolf5]', x=-11087.48, y=69.00, z=-7319.63], class_3222['Tamburin771'/71780, l='ServerLevel[Zunwolf5]', x=13440.70, y=142.00, z=12641.96], class_3222['LocalTrashClown'/84199, l='ServerLevel[Zunwolf5]', x=-502.14, y=64.00, z=322.29]]
	Data Packs: vanilla, Fabric Mods, file/GiantOrigin-1.19v1.zip (incompatible), file/TadpolefrogDontUnzip.zip (incompatible), file/Pocket_Dragon_V7.zip, file/Hellchained_v0.4.1.zip, file/Explorer.zip (incompatible), file/VanillaTweaks_c702835.zip, file/afk display v1.1.3.zip, file/cauldron concrete v2.0.6.zip, file/larger phantoms v1.2.4.zip, file/track raw statistics v1.5.3.zip, file/track statistics v1.1.4.zip, file/Sculkborn_1.3.zip, file/Arachnid.zip (incompatible), file/FutaraDragon_Origin_v21_MC_1_19.zip, file/MonkeyOrigin-1.19v3.zip (incompatible)
	World Generation: Stable
	Is Modded: Definitely; Server brand changed to 'fabric'
	Type: Dedicated Server (map_server.txt)
```

**Comment by quiqueck** [comment]
Thanks for reporting. While I can not reproduce the crash, I implemented something that may fix the issue. 

Upon reading the stack trace again, I noticed that it looks like carped mods `activateWithOptionalCactus` was used. Is that right?

**Comment by AyMeow** [comment]
> 感谢您的报告。虽然我无法重现崩溃，但我实施了一些可能解决问题的方法。
> 
> 再次阅读堆栈跟踪后，我注意到它看起来像是`activateWithOptionalCactus`使用了 carped mods。那正确吗？

Hi! I also encountered this problem! I did install the carpet mod. Is the carpet mod incompatible with betterend? This is my error report:https://mclo.gs/zzzYiju

**Comment by quiqueck** [comment]
Should be fixed in 2.0.7
